c10/util/Logging.cpp

```diff
@@ -198,23 +198,42 @@ bool IsGoogleLoggingInitialized();
 } // namespace google
 
 namespace c10 {
-bool InitCaffeLogging(int* argc, char** argv) {
-  if (*argc == 0)
-    return true;
+namespace {
+
+void initGoogleLogging(char const* name) {
 #if !defined(_MSC_VER)
   // This trick can only be used on UNIX platforms
   if (!::google::glog_internal_namespace_::IsGoogleLoggingInitialized())
 #endif
   {
-    ::google::InitGoogleLogging(argv[0]);
+    ::google::InitGoogleLogging(name);
 #if !defined(_MSC_VER)
     // This is never defined on Windows
 #if !defined(__XROS__)
     ::google::InstallFailureSignalHandler();
 #endif
 #endif
   }
+
   UpdateLoggingLevelsFromFlags();
+}
+
+} // namespace
+
+void initLogging() {
```

Comment:
nit: maybe `initTorchLogging` to distinguish it from caffe2? Although maybe not because `InitCaffeLogging` may eventually be deprecated.

Reply:
Yes, that's why I kept it `initLogging()`. If in the future `InitCaffeLogging` eventually gets deleted, having `Torch` in its name might sound redundant.